docs/cloud/http-api.md

```diff
@@ -182,6 +182,25 @@ So if the delete query matches a branch, **every** series under that branch will
 curl -u "api_key:<Your Grafana.com API Key>" https://<tsdbgw>/metrics/delete -d query=some.series.to.delete.*
 ```
 
+#### Tagged With `/tags/delSeries`
```

Comment:
I think this might actually work with untagged series as well?

Not sure if we would want to document that at all, or if that would add confusion.

Reply:
It wouldn't work for untagged metrics, because they're indexed in different data structures than this modifies

Reply:
Cool, I'm just wondering if that's a difference/regression from the graphite API.


Can't find anything definitive, just a single [test](https://github.com/graphite-project/graphite-web/blob/1527eb4dcff3c1891b0850d2d2f3afac1f31e43e/webapp/tests/test_tags.py#L218-L222) that uses a metric without a tag.

(`/tag/delSeries` calls `del_multi_series` [here](https://github.com/graphite-project/graphite-web/blob/87240bf6d2079691dcfb16616565b6247648ddf2/webapp/graphite/tags/views.py#L57), which calls `del_series` [here](https://github.com/graphite-project/graphite-web/blob/1527eb4dcff3c1891b0850d2d2f3afac1f31e43e/webapp/graphite/tags/base.py#L167))

Reply:
in graphite you couldn't remove an untagged serie with this call afaik. actually the way how the tags in graphite work are a bit different, because when carbon/whisper store the tagged metrics they store a normal whisper file but it's name is something like a hash which doesn't show up in the normal graphite metrics explorer, then carbon does an http call to graphite to enter this metric name & tags into the tagdb. once they're in the tagdb they can be found with `seriesByTags()`. When calling `/tags/delSeries` they get removed again from the tagdb, while the whisper files which are the actual series themselves remain on the disk, so they become non-retrievable even though they don't get deleted from the disk.

Just to be sure i tried that:

```
$ curl 'http://localhost:8000/metrics/find?query=some.id.of.a.metric.1' | jq
[
  {
    "text": "1",
    "expandable": 0,
    "leaf": 1,
    "id": "some.id.of.a.metric.1",
    "allowChildren": 0
  }
]
$ curl http://localhost:8000/tags/delSeries -d path=some.id.of.a.metric.1
true
$ curl 'http://localhost:8000/metrics/find?query=some.id.of.a.metric.1' | jq
[
  {
    "text": "1",
    "expandable": 0,
    "leaf": 1,
    "id": "some.id.of.a.metric.1",
    "allowChildren": 0
  }
]
$ curl http://localhost:8000/tags/delSeries -d path=idonotexist
true
```

Reply:
I like how it returns true to confirm that it worked 🙃